server/routes/visitJourney/visitType.test.ts

```diff
@@ -0,0 +1,184 @@
+import type { Express } from 'express'
```

Comment:
Added with a skip, but would like the others merging before I try and separate this out. Can remove this from the commit if necessary.